dashboard/app/assets/stylesheets/application.scss

```diff
@@ -2241,6 +2209,11 @@ a.download-video {
   border: 1px solid $lighter_gray;
 }
 
+.curriculum-reference #iframe-loading.loading {
+  height: 200px;
+  background: image-url("spinner-big.gif") no-repeat center center;
+}
```

Comment:
This fixes this instance of the loading class: https://github.com/code-dot-org/code-dot-org/blob/48a5dd9cbdd8a7e3f5cbf426c782d4b15242fa56/dashboard/app/views/levels/_curriculum_reference.haml#L8